api/v1/schemas/token.py

```diff
@@ -9,7 +9,7 @@ class Token(BaseModel):
     token_type: str
 
 class TokenData(BaseModel):
-    username: Optional[str] = None
+    user_id: Optional[str] = None
```

Comment:
you edited this schema and I don't see you making use of it

Reply:
Yes the schema had already been implemented in the user service, I can remove the schema in another commit